src/App.jsx

```diff
@@ -16,6 +23,10 @@ const Registration = lazy(() => import('./components/Registration'));
 const Login = lazy(() => import('./components/Login'));
 
 function App() {
```

Comment:
Major:
App component renders 2 times on initial step.
This causes double hook runs => 2 calls to backend (Network tab in browser dev tools).
Root cause should be found and fixed.